src/witan/send/driver/validate/costs.clj

```diff
@@ -50,3 +50,11 @@
   (into #{}
         (remove (partial transition-has-cost? costs))
         transitions))
+
+(defn build-dummy-costs [needs settings census-data]
```

Comment:
would be good to have one where you pass in just census-data and it generated the needs and settings and another that took needs and settings, but didn't need the currently unused census data